README.md

```diff
@@ -1,12 +1,14 @@
-# Shopsys Project-base
+# Shopsys Framework Demoshop
 
-This repository is part of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
+This repository is Demoshop project of Shopsys Framework - a **scalable ecommerce framework** for fast-growing ecommerce sites created and maintained by in-house developers or outsourcing agencies.
 
-Shopsys/project-base repository contains the fully ready base for building your Shopsys Framework projects. This repository is READ-ONLY and it is maintained by monorepo [shopsys/shopsys](https://github.com/shopsys/shopsys).
+Shopsys/demoshop repository contains demostration project to show how you can build your own project on top of Shopsys Framework. 
```

Comment:
Shopsys/demoshop feels wierd to me, because you use captions while speaking about package name. I think that it should be like this `shopsys/demoshop` or `Shopsys Demoshop`